src/features/movies/MovieList/index.js

```diff
@@ -41,14 +46,20 @@ function MovieList() {
 		history.push(`${location.pathname}?page=${currentPage}`);
 	}, [currentPage]);
 
```

Comment:
i think you should add an useEffect here that clears the inputValue of SearchSlice, on render

```
	useEffect(() => {
		dispatch(setInputValue(``));
	}, []);
```